packages/http/src/validator/index.ts

```diff
@@ -186,7 +200,7 @@ export const validateOutput: ValidatorFn<IHttpOperation, IHttpResponse> = ({ res
           )
         ),
         validateBody(element.body, response.contents || [], ValidationContext.Output, mediaType, bundle),
-        validateHeaders(element.headers || {}, response.headers || [], bundle)
+        validateHeaders(element.headers || {}, response.headers || [], ValidationContext.Output, bundle)
```

Comment:
was this an optional parameter? can we remove its optionality?

Reply:
It wasn't a parameter to the header validation function before at all. I add it in this PR and it is not optional. It was only being passed to the body validator function before for some reason, and not the other validators 🤷‍♀️

Reply:
i see it now.